app/src/main/res/values-in-rID/strings.xml

```diff
@@ -0,0 +1,168 @@
+  <string name="human_heartbeat">Detak jantung manusia</string>
+  <string name="light_rain">Hujan ringan</string>
+  <string name="moderate_rain">Hujan sedang</string>
+  <string name="morning_in_a_village">Pagi di desa</string>
+  <string name="moving_train">Kereta bergerak</string>
+  <string name="night">Malam</string>
+  <string name="office">Kantor</string>
+  <string name="pink_noise">Pink noise</string>
+  <string name="public_library">Perpustakaan umum</string>
+  <string name="purring_cat">Mendengkur kucing</string>
+  <string name="quiet_conversation">Percakapan senyap</string>
+  <string name="rolling_thunder">Gemuruh bergulir</string>
+  <string name="screeching_seagulls">Burung camar yang melengking</string>
+  <string name="seaside">Tepi laut</string>
+  <string name="soft_wind">Angin lembut</string>
+  <string name="thunder_crack">Gemuruh retak</string>
+  <string name="train_horn">Klakson kereta</string>
+  <string name="walking_through_the_snow">Berjalan melintasi salju</string>
+  <string name="water_hose">Selang air</string>
+  <string name="water_stream">Aliran air</string>
+  <string name="white_noise">White noise</string>
+  <string name="wind_in_palm_trees">Angin di pohon palem</string>
+  <string name="wind_in_chimes_of_shells">Angin di lonceng kerang</string>
+  <string name="app_description">Santai dan tingkatkan produktivitas dengan kebisingan latar belakang minimal.</string>
+  <string name="connect_with_author">Hubungkan dengan Penulis</string>
+  <string name="about__email">Email</string>
+  <string name="about__github">Github</string>
+  <string name="about__twitter">Twitter</string>
+  <string name="credits">Kredit</string>
+  <string name="dismiss">Singkirkan</string>
+  <string name="save_preset">Simpan Preset</string>
+  <string name="name">Nama</string>
+  <string name="save">Simpan</string>
+  <string name="cancel">Batal</string>
+  <string name="preset_name_cannot_be_empty">Nama preset tidak boleh kosong!</string>
+  <string name="preset_already_exists">Preset dengan nama yang diberikan sudah ada!</string>
+  <string name="preset_saved">Preset disimpan!</string>
+  <string name="saved_presets">Preset Tersimpan</string>
+  <string name="preset_delete_confirmation">Konfirmasi hapus% s?</string>
+  <string name="delete">Hapus</string>
+  <string name="preset_deleted">Preset dihapus!</string>
+  <string name="preset_info__description">
+    Anda belum membuat preset apa pun. Untuk membuat preset, buka Perpustakaan dan putar suara.
+    Kemudian klik tombol Simpan yang muncul di sudut bawah layar.
+</string>
+  <string name="sleep_timer">Pengatur Waktu Tidur</string>
+  <string name="sleep_timer_description">
+    Noice akan mati sendiri ketika timer berakhir. Gunakan tombol di bawah untuk menambahkan waktu dan memulai
+    pengatur waktu.
+</string>
+  <string name="reset">Setel ulang</string>
+  <string name="auto_sleep_schedule_cancelled">Pengatur waktu tidur otomatis dibatalkan!</string>
```

Comment:
Google translates it to _Sleep timer automatically canceled!_ but it needs to imply _Sleep timer cancelled!_ or _Automatic sleep timer cancelled!_